hoodie-common/src/main/java/com/uber/hoodie/common/table/log/HoodieLogFormat.java

```diff
@@ -206,6 +219,13 @@ static WriterBuilder newWriterBuilder() {
     return new HoodieLogFileReader(fs, logFile, readerSchema, HoodieLogFileReader.DEFAULT_BUFFER_SIZE, false, false);
   }
 
+  static HoodieLogFormat.Reader newReader(FileSystem fs, HoodieLogFile logFile, Schema readerSchema, boolean
+      readBlockLazily, boolean reverseReader)
+      throws IOException {
+    return new HoodieLogFileReader(fs, logFile, readerSchema, HoodieLogFileReader.DEFAULT_BUFFER_SIZE,
```

Comment:
Question : Does the buffer size needs to be configurable by user ?

Reply:
Right now, we don't change it even for compaction jobs but it is exposed as a config..

Reply:
So, this particular method is so that we can override certain parameters like reverseReading for custom use-cases which will not be part of the config..